common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -1,4 +1,4 @@
-(function (define) {
+(function (define, Draggabilly) {
```

Comment:
I don't know a *ton* about Javascript, but a number of acceptance tests are failing with [Error loading dependencies: type=scripterror modules=[u'video/09_video_caption.js']](https://build.testeng.edx.org/job/edx-platform-all-tests-pr/79/SHARD=1,TEST_SUITE=cms-acceptance,label_exp=jenkins-worker/testReport/junit/CMS/Component%20Adding%20_%20I%20can%20add%20Common%20Problem%20components/Given_I_am_in_Studio_editing_a_new_unit/)

@singingwolfboy do you know what this means? Is this a JS compilation error or something?